app/Http/Controllers/Api/AccessoriesController.php

```diff
@@ -197,28 +200,23 @@ public function checkedout($id, Request $request)
         $offset = request('offset', 0);
         $limit = request('limit', 50);
 
-        $accessory_users = $accessory->users;
-        $total = $accessory_users->count();
+        $accessory_checkouts = $accessory->checkouts;
+        $total = $accessory_checkouts->count();
 
         if ($total < $offset) {
             $offset = 0;
         }
 
-        $accessory_users = $accessory->users()->skip($offset)->take($limit)->get();
+        $accessory_checkouts = $accessory->checkouts()->skip($offset)->take($limit)->get();
 
         if ($request->filled('search')) {
-            $accessory_users = $accessory->users()
```

Comment:
Why was this removed? This allows searching on user's names in the checkout list.

Reply:
I see, you added a query scope further down the line.

Reply:
This has been replaced with a search over all three kinds of models depending of the checkout type (see advancedTextSearch of the new AccessoryCheckout model).